docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -184,27 +188,27 @@ Here you can set the maximum transmission unit (MTU) of the VPN interface. The r
 
 #### Restore VPN automatically
 
-If enabled, this feature automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
+This option automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
 
 #### Packet capture (PCAP)
 
-If enabled, AdGuard will create the special file name `timestamp.pcap` (for instance, 1682599851461.pcap). It contains all network packets transferred through the VPN. This file is located in the app cache directory and can be analyzed with the Wireshark program.
+This option makes AdGuard create a file `timestamp.pcap` (for instance, 1682599851461.pcap) that lists all network packets transferred through the VPN and is located in the app cache directory. This file can be analyzed with the Wireshark program.
```

Comment:
If this is enabled, AdGuard will create a file `timestamp.pcap` (for instance, 1682599851461.pcap) in the app cache directory. This file lists all network packets transferred through the VPN and can be analyzed with the Wireshark program.